internal/webhook/certs/cert.go

```diff
@@ -29,7 +28,7 @@ const (
 	DefaultCertDir = "/tmp/k8s-webhook-server/serving-certs"
 )
 
-func SetupCertSecret(ctx context.Context, secretName, secretNamespace, serviceName string, logger *zap.SugaredLogger) error {
+func SetupCertSecret(ctx context.Context, secretName, secretNamespace, serviceName, deployName, addOwnerRef string, logger *zap.SugaredLogger) error {
```

Comment:
We can pack those configuration params into WebhookConfig
`secretName, secretNamespace, serviceName, deployName, addOwnerRef ` and pass it also to `controller.go`
This struct can look like that:
```
type CertConfig struct {
	ServiceName      string
	ServiceNamespace string
	DeployName       string
	AddOwnerRef      bool
}
```